tools/pages/admin/admin-management/types.ts

```diff
@@ -0,0 +1,10 @@
+import { Role } from '@lib/types';
+
+export type EmployeeData = {
```

Comment:
Add a comment, especially here because the variable `EmployeeData` is very vague and carries no context with it